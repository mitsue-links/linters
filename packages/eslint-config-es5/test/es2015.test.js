/* eslint strict: [2, "global"] */
'use strict';

const path = require('path');
const {lintFile} = require('./helper');

process.chdir(__dirname);

describe('ECMAScript 2015', () => {
    test('es2015_fail.js', async () => {
        const filePath = 'es2015_fail.js';
        const data = await lintFile(path.join('fixtures', filePath));

        expect(data).toMatchSnapshot();
    });

    test('es2015_error.js', async () => {
        const filePath = 'es2015_error.js';
        const data = await lintFile(path.join('fixtures', filePath));

        expect(data).toMatchSnapshot();
    });
});
