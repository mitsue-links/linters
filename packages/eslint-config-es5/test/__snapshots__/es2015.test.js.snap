// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ECMAScript 2015 es2015_error.js 1`] = `
Array [
  Object {
    "filePath": "es2015_error.js",
    "messages": Array [
      Object {
        "line": 2,
        "ruleId": "no-undef",
        "severity": 2,
      },
    ],
  },
]
`;

exports[`ECMAScript 2015 es2015_fail.js 1`] = `
Array [
  Object {
    "filePath": "es2015_fail.js",
    "messages": Array [
      Object {
        "line": 1,
        "ruleId": "Parsing error: The keyword 'const' is reserved",
        "severity": 2,
      },
    ],
  },
]
`;
